donate/settings/staging.py

```diff
@@ -0,0 +1,20 @@
+from .environment import env
+from .base import Base
+from .secure import Secure
+from .database import Database
+from .redis import Redis
+from .s3 import S3
+from .braintree import Braintree
+from .sentry import Sentry
+
+
+class Staging(Base, Secure, Database, Redis, S3, Braintree, Sentry):
+
+    SECRET_KEY = env('DJANGO_SECRET_KEY')
+    BASKET_API_ROOT_URL = env('BASKET_API_ROOT_URL')
+    BASKET_SQS_QUEUE_URL = env('BASKET_SQS_QUEUE_URL')
+
+    @classmethod
+    def pre_setup(cls):
+        cls.LOGGING['loggers']['rq.worker']['handlers'] = ['DEBUG']
+        cls.LOGGING['loggers']['rq.worker']['level'] = 'DEBUG'
```

Comment:
@alanmoo FYI this is what triggers worker queue logs on staging, but not on production